pgsql.hpp

```diff
@@ -25,10 +25,10 @@ void escape(const std::string &src, std::string& dst);
 
 
 inline void pgsql_CopyData(const char *context, PGconn *sql_conn, const char *sql) {
-    pgsql_CopyData(context, sql_conn, sql, strlen(sql));
+    pgsql_CopyData(context, sql_conn, sql, (int) strlen(sql));
```

Comment:
the length goes to `PQputCopyData(PGconn *conn, const char *buffer, int nbytes);` so we're fine